src/index.html

```diff
@@ -40,9 +48,16 @@
       <manifold-configured-feature label="crackers" value="true"></manifold-configured-feature>
       <manifold-configured-feature label="juice" value="60"></manifold-configured-feature>
     </manifold-subscription-create>
+
+    <h2>List subscriptions</h2>
     <manifold-subscription-list
       heading="Your Subscriptions"
       owner="fake-owner-id"
     ></manifold-subscription-list>
+
+    <h2>View subscription details</h2>
+    <manifold-subscription-details
+      id="2475k59d5tbk57p95v5w8x4rb4mnp"
```

Comment:
where `id` is an existing html attribute maybe we should use. `subscription-id`

Reply:
ha trudat